docs/src/pages/components/dialogs/SimpleDialog.js

```diff
@@ -63,9 +63,8 @@ function SimpleDialog(props) {
 }
 
 SimpleDialog.propTypes = {
-  onClose: PropTypes.func,
-  open: PropTypes.bool,
```

Comment:
It's because it's spread onto the dialog, that counts as "unused" and it gets omitted
https://github.com/mui-org/material-ui/blob/2b90289632c628dfa96e5c968b98258352e0c445/docs/src/pages/components/dialogs/SimpleDialog.js#L38

Reply:
Can we configure that? "unused" isn't really accurate.

Reply:
Yes
https://github.com/merceyz/typescript-to-proptypes/blob/232634f7782a8119d012ceb69b27e30911ff8052/src/injector.ts#L6-L12

Reply:
The reason it does this is if you extend an interface like `ButtonProps` but you only use `color` directly and spread the rest down to `button`, it shouldn't include all PropTypes from `ButtonProps`

Reply:
I updated the demo to use it directly